changelog.txt

```diff
@@ -1,3 +1,16 @@
+2017.09.25 Version 0.6.2
+ * All Services
+   - Upgrade to Netstandard 2.0
+   - Consistent the TransferException format, now all TransferException with StorageException will have StorageException as inner exception.
```

Comment:
"Consistent" =>?

Reply:
How about: Make the TransferException format consistent